### DISCUSSION TITLE
Begin X.X/End X.X in book

### DISCUSSION THREAD

**Discussion by rickhille** [IntelliTect/EssentialCSharp.Web#636] [Feedback and Ideas]
Hello. I bought a copy of Essential C# 12.0 and I just started reading it. I am now in Chapter 1 and one thing that I was not able to figure out is what are those "Begin X.X/End X.X" boxes on the side. I thought I might have missed the explanation of those Symbols in the previous pages but I must be completely blind because I could not find it... It looks like those are not on the Website either. Could you please give me an explanation? Thanks, Richard

**Comment by BenjaminMichaelis** [comment]
> Hello. I bought a copy of Essential C# 12.0 and I just started reading it. I am now in Chapter 1 and one thing that I was not able to figure out is what are those "Begin X.X/End X.X" boxes on the side. I thought I might have missed the explanation of those Symbols in the previous pages but I must be completely blind because I could not find it... It looks like those are not on the Website either. Could you please give me an explanation? Thanks, Richard

Hi! An explanation should be in the frontmatter content, but those are describing the version of C# where the feature began in. So if a topic (feature) began in C# 7.0, it would be between a "Begin X.X" and then later a "End X.X". Does this clarify it?
